src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -591,9 +630,10 @@ private void sendHubMessage(HubMessage message) {
         } else {
             logger.debug("Sending {} message.", message.getMessageType().name());
         }
-        transport.send(serializedMessage).subscribeWith(CompletableSubject.create());
+        Completable sendCompletable = transport.send(serializedMessage).subscribeWith(CompletableSubject.create());
 
         resetKeepAlive();
+        return sendCompletable;
```

Comment:
It seems most callers don't use this return value. Is this needed for `Observable.subscribe`? If so, why?

Reply:
This isn't needed actually. this was for a thought I had initially, but I went a different direction and forgot to revert this.